testData/preferences/manuel.json5

```diff
@@ -61,14 +61,6 @@
                         ],
                         "value": 1024
                     }
-                ],
-                "conditions": [
```

Comment:
The comment at line 17, 18 explaining the condition change should be removed. Thanks.

Reply:
Done